server/controllers/resetController.js

```diff
@@ -0,0 +1,52 @@
+import { config } from 'dotenv';
+import authHelper from '../helpers/authHelpers';
+import models from '../sequelize/models';
+
+config();
+const { User } = models;
+class ResetController {
+
+  /**This method is for to handle the request to signup
```

Comment:
Consider updating your comment. The method sends a password reset link instead.

Reply:
Done